medusa/server/api/v2/show.py

```diff
@@ -86,6 +86,8 @@ def _handle_detailed_show(self, tv_show, query):
                 new, existing = tv_show.get_backlogged_episodes(allowed_qualities=allowed_qualities,
                                                                 preferred_qualities=preferred_qualities)
                 data = {'new': new, 'existing': existing}
+            elif query == 'archiveEpisodes':
```

Comment:
Oh this looked like you were using this TO archive the files.
I need to stop reviewing shit when I've just woken up.

Use true/false not 1/0 or True/False since true/false are booleans in js. This should work like  how passing an int turns into a js number(remember the parseInt thing).